etc/android-studio.profile

```diff
@@ -1,14 +1,12 @@
-# Persistent global definitions go here
+# Firejail profile for android-studio
+# This file is overwritten after every install/update
+# Persistent global definitions
 include /etc/firejail/globals.local
-
-# This file is overwritten during software install.
-# Persistent customizations should go in a .local file.
+# Persistent local customizations
 include /etc/firejail/android-studio.local
 
-# Firejail profile for Android Studio
-
-noblacklist ${HOME}/.AndroidStudio*
 noblacklist ${HOME}/.android
+noblacklist ${HOME}/.AndroidStudio*
```

Comment:
That is just Java's arraylist.sort(String::compareToIgnoreCase); Is it necessary?